astronomer/airflow/version_check/templates/update-available.html

```diff
@@ -1,27 +1,59 @@
-{% if current_user.is_authenticated and cea_update_available %}
-  {% set level = "error" if cea_update_available.level == "critical" else "warning" %}
-  <div class="alert alert-{{ level }} ac-update-notice">
-    {% if "can_dismiss" | is_item_visible("UpdateAvailable") %}
-      <button data-href="{{ url_for('UpdateAvailable.dismiss', version=cea_update_available.version) }}" title="Ignore this update, and hide this message from all users">&times; Ignore this update</button>
+{% if current_user.is_authenticated %}
+  {% if cea_update_available.update_notice %}
+    {% set update_level = "warning" %}
+    {% if cea_update_available.update_notice.level == "critical" %}
+      {% set update_level = "error" %}
     {% endif %}
-    <p><strong>A new version of {{cea_update_available.app_name}} is available.</strong></p>
-    <p>
-    {% if cea_update_available.url %}
-      <a href="{{ cea_update_available.url }}">
+    <div class="alert alert-{{ update_level }} ac-update-notice">
+      {% if "can_dismiss" | is_item_visible("UpdateAvailable") %}
+        <button data-href="{{ url_for('UpdateAvailable.dismiss', version=cea_update_available.update_notice.version) }}" title="Ignore this update, and hide this message from all users">&times; Ignore this update</button>
+      {% endif %}
+      <p><strong>A new version of {{ cea_update_available.update_notice.app_name }} is available.</strong></p>
+      <p>
+        {% if cea_update_available.update_notice.url %}
+          <a href="{{ cea_update_available.update_notice.url }}">
+        {% endif %}
+          Version {{ cea_update_available.update_notice.version }}
+        {% if cea_update_available.update_notice.url %}
+          </a>
+        {% endif %}
+        was released on
+        <time datetime="{{ cea_update_available.update_notice.date_released.isoformat() }}">{{ cea_update_available.update_notice.date_released.strftime("%Y-%m-%d") }}</time>
+        {%- if cea_update_available.update_notice.level == "critical" %}
+          and this is a critical fix
+        {%- endif -%}
+        .
+      </p>
+      {% if cea_update_available.update_notice.description %}
+        <p>{{ cea_update_available.update_notice.description }}</p>
+      {% endif %}
+    </div>
+  {% endif %}
+
+  {% if cea_update_available.eol_notice %}
+    {% set eol_level = "warning" if cea_update_available.eol_notice.days_to_eol > 0 else "error" %}
```

Comment:
We already do this in python. We should do it in 1 place.

Reply:
Added the logic to python now